src/LibraryView.tsx

```diff
@@ -61,6 +63,17 @@ export class LibraryView {
                 thisObject.layoutSpecsJson = JSON.parse(jsonString);
                 thisObject.updateContentsInternal();
             });
+
+        // fetch search items (in json format) if queryString exists and is not empty
+        if (queryString && queryString.length > 0) {
+            fetch(queryString)
+                .then(function (response: Response) {
+                    return response.text();
+                }).then(function (jsonString) {
+                    thisObject.searchItemsJson = JSON.parse(jsonString);
+                    thisObject.updateContentsInternal();
+                });
+        }
```

Comment:
This doesn't belong here, I think you have misunderstood the purpose of `searchEndpoint`, it should be a fully qualified URL to a search REST API. E.g. `https://dynamoreach.com/search-library` and when user types in search keyword, we will then call it like so:

```
https://dynamoreach.com/search-library?query=PointByCoordinates
```

Does that make sense?